modules/migration-understanding-cam.adoc

```diff
@@ -10,23 +10,26 @@ The Cluster Application Migration (CAM) tool enables you to migrate Kubernetes r
 
 Migrating an application with the CAM web console involves the following steps:
 
-. Installing the CAM Operator on the source and target clusters
+. Installing the CAM Operator on all clusters
 +
 [NOTE]
 ====
-The CAM Operator installs the CAM web console on the target cluster by default.
+The CAM Operator installs the CAM tool (CAM web console and Migration controller) on the target cluster by default. You can change this configuration in order to install the CAM tool on another cluster.
+
+If the CAM is not installed on the target cluster, you must configure cross-origin resource sharing on the target cluster and add the target cluster to the CAM web console.
```

Comment:
Should "the CAM" be "the CAM tool" or something else?

Reply:
Yes, good catch.